app/views/posts/index.html.erb

```diff
@@ -1,11 +1,50 @@
-<h1>Posts</h1>
+<%= stylesheet_link_tag 'posts_index' %>
 
+<div class="user-show-container">
+  <div class="user-image-container">
+    <img src="<%= @user.photo %>" alt="<%= @user.name %>">
+  </div>
+
+  <div class="user-details-container">
+    <h1><%= @user.name %></h1>
+    <p class="post-count">Number of posts: <%= @user.posts.count %></p>
+  </div>
+</div>
+
+
+<div class="user-posts-container">
 <ul>
-  <% @posts.each do |post| %>
-    <li>
-      <%= link_to post.title, user_post_path(@user, post) %>
-    </li>
+
+<% @user.posts.each do |post| %>
```

Comment:
- [ ] By now as specified above, you should have a method in your models that permit you to render the user's 5 most recent comments of a post instead of displaying all the posts. Please kindly look into this so that we can limit the number of comments on this page.